todoapp/src/components/TodoForm.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { defineProps, defineEmits, ref } from 'vue'
+
+const props = defineProps(['todos', 'inputContent'])
+const emits = defineEmits(['addTodo'])
+const inputContentRef = ref(props.inputContent)
```

Comment:
Please create a v-model instead of using the inputContent prop and the ref, you can create a writable computed for this purpose: 
define the appropriate props and events, and then make this style of writable computed:

const inputValue = computed({
  get() {
    return props.modelValue
  },
  set(newValue) {
    emit('update:modelValue', newValue)
  }
})